lib/LedgerSMB/Scripts/setup.pm

```diff
@@ -838,11 +865,17 @@ sub fix_tests{
     my $table = $check->table;
     my $where = $check->id_where;
     my @edits = @{$check->columns};
+    # If we are inserting and id is displayed, we want to insert
+    # at this exact location
```

Comment:
We need to address 2 situations in inserting:

1. I need data in this table, please provide some. For this, there is no key to display

2. I have found referals to a missing entry in this table, here's what I got, please complete. For this, I show the referal key